src/pages/home/index.tsx

```diff
@@ -1,11 +1,20 @@
 import scss from './index.module.scss';
 
+import CharacterCountdown from '../../components/CharacterCountDown';
+import Tabber from '../../components/Tabber';
+
 const Home = () => {
   return (
-    <div className={scss.Home}>
-      <h1>Octopus</h1>
+    <div>
+      <div className={scss.Home}>
+        <h1>Octopus</h1>
+        <img className={scss.Logo} src="/logo.png" />
 
-      <img className={scss.Logo} src="/logo.png" />
+        <div>
+          <CharacterCountdown maxLength={140} />
+        </div>
+      </div>
+      <Tabber></Tabber>
```

Comment:
Esse componente não tem children, pode ser chamado assim.
```suggestion
      <Tabber />
```